code/modules/mob/living/living_defense.dm

```diff
@@ -268,6 +268,10 @@
 		return FALSE
 	if(user.pull_force < move_force)
 		return FALSE
+	// This if-statement checks if the user is horizontal, and if the user either has no martial art, or has judo, drunk fighting or krav, in which case it should also fail
+	if(IS_HORIZONTAL(user) && (!user.mind.martial_art || is_type_in_list(user.mind.martial_art, list(/datum/martial_art/judo, /datum/martial_art/drunk_brawling, /datum/martial_art/krav_maga))))
```

Comment:
Personally, I feel like this should just be a var on the martial art

Reply:
agree

Reply:
Done!